arbitrum-docs/node-running/how-tos/data-availability-committee/partials/parameters/_local-files-parameters.mdx

```diff
@@ -1,3 +1,9 @@
+::::warning Create the local directory before launching your DAS
+
+Make sure you create the directory specified in `local-file-storage.data-dir` before launching your DAS to avoid potential permission issues with Docker or Kubernetes.
+
+::::
```

Comment:
Really nice to add partials to warn readers across relevant articles!
I'm starting to contemplate how we could manage partial indexation. Their number is increasing.